lib/Tags/TagsPane.test.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import user from '@testing-library/user-event';
+
+import TagsPane from './TagsPane';
+
+const TAGS = [{ id: 'tag1', label: 'tag-1' }];
+
+const renderComponent = (props = {}) => (render(
+  <TagsPane
+    {...props}
+  />,
+));
+
+describe('TagsPane', () => {
+  const mutator = {
+    tagsPaneTags: {
+      GET: jest.fn().mockResolvedValue(TAGS),
+      POST: jest.fn().mockResolvedValue(),
+    },
+  };
+  const entity = { tags: { tagList: [] } };
+  const updateEntity = jest.fn();
+  const onClose = jest.fn();
+
+  beforeEach(() => {
+    updateEntity.mockClear();
+    onClose.mockClear();
+    mutator.tagsPaneTags.POST.mockClear();
+  });
+
+  it('should display loading', async () => {
+    const { container } = renderComponent({ entity, mutator, updateEntity, onClose });
+
+    await waitFor(() => expect(container.querySelector('.spinner')).toBeDefined());
+  });
+
+  it('should call update entity on unassign tag', async () => {
+    renderComponent({ entity: { tags: { tagList: ['tag-1'] } }, mutator, updateEntity, onClose });
+    await waitFor(() => expect(screen.getByText('1 item selected')).toBeDefined());
```

Comment:
I don't think you need expect here, you shouldn't have any code after expectations

Reply:
we need to wait until all is loaded. I don't know what might be better to check.